.github/workflows/macos_publish_dmg_release.yml

```diff
@@ -286,14 +286,13 @@ jobs:
             echo "FILES_UPLOADED='No files uploaded.'" >> $GITHUB_ENV
           fi
 
-      - name: Send release annotation pixel
-        if: github.event.inputs.release-type != null && github.event.inputs.release-type != 'internal'
+      - name: Send Release Pixel
+        if: success() && github.event.inputs.release-type != null && github.event.inputs.release-type != 'internal'
```

Comment:
What I wonder is if we want to have it for internals as well:
1/ we can have only for public
2/ we can have for all
3/ we can have two pixels, one for internal and one for public releases

I’m sort of leaning towards 3, as it gives us most flexibility around how to use these pixels, but it’s a bt more tricky to send (e.g. we’d have internal, that then gets promoted to public).

Reply:
Yeah good question. If we do want internal pixels (and we may as well), then I agree option 3 is suitable. I think this will be doable by keeping this pixel as-is as the final release, but updating the other workflow file to send the internal pixel - let me take a go at that on Thursday.